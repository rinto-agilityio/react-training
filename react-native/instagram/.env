PERSIST_STORE_KEY=instagram

FIREBASE_API_KEY=AIzaSyCZpGh86wxsZM8H_PyZVKsk_MnjopqYDyY
FIREBASE_AUTH_DOMAIN=instagram-rn.firebaseapp.com
FIREBASE_DATABASE_URL=https://instagram-rn.firebaseio.com
FIREBASE_PROJECT_ID=instagram-rn
FIREBASE_STORAGE_BUCKET=gs://instagram-rn.appspot.com
FIREBASE_MESSAGING_SENDER_ID=201897093582
