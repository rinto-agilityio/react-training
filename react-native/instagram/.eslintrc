{
  "plugins": ["react"],
  "extends": ["eslint:recommended", "roc", "eslint:all", "plugin:react/all"],
  "rules": {
    "semi": ["error", "always"],
    "quotes": ["error", "single"],
    "comma-dangle": [
      "error",
      {
        "arrays": "never",
        "objects": "never",
        "imports": "never",
        "exports": "never",
        "functions": "ignore"
      }
    ],
    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
  }
}
