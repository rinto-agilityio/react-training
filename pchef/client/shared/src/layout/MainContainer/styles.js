// Lib
import { StyleSheet } from 'react-native'

// Themes
import { METRICS } from '../../themes'

const styles = StyleSheet.create({
  webContainer: {
    width: METRICS.extraLargeScreen,
    marginTop: METRICS.resetMargin,
    marginBottom: METRICS.resetMargin,
    marginRight: 'auto',
    marginLeft: 'auto',
    zIndex: METRICS.iconZindex,
  },
  mobileContainer: {},
})

export default styles
