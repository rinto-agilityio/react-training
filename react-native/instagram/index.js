import { AppRegistry } from "react-native";
import App from "./src/App";

AppRegistry.registerComponent("instagram", () => App);
