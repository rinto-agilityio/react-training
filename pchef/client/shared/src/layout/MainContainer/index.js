// Libs
import * as React from 'react'
import { View } from 'react-native'

// Styles
import styles from './styles'

type Props = {
  children: React.Node,
  type: string,
}

const MainContainer = ({ children, type }: Props) => (
  <View style={[type === 'web' ? styles.webContainer : styles.mobileContainer]}>
    { children }
  </View>
)

export default MainContainer
