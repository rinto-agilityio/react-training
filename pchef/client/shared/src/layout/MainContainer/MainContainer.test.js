// Components
import { Text } from 'react-native'
import MainContainer from '.'

describe('MainContainer', () => {
  it('should render type web', () => {
    const component = renderer
      .create(
        <MainContainer type="web">
          <Text>test</Text>
        </MainContainer>
      )
      .toJSON()
    expect(component).toMatchSnapshot()
  })

  it('should render type mobile', () => {
    const component = renderer
      .create(
        <MainContainer type="mobile">
          <Text>test</Text>
        </MainContainer>
      )
      .toJSON()
    expect(component).toMatchSnapshot()
  })
})
