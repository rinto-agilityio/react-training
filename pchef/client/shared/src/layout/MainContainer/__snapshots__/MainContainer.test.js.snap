// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer Should render type mobile 1`] = `
<div
  className="css-view-1dbjc4n"
>
  <div
    className="css-text-901oao"
    dir="auto"
  >
    test
  </div>
</div>
`;

exports[`MainContainer Should render type web 1`] = `
<div
  className="css-view-1dbjc4n r-marginBottom-p1pxzi r-marginLeft-1jj8364 r-marginRight-lchren r-marginTop-1mnahxq r-width-16k0tzm r-zIndex-1wyyakw"
>
  <div
    className="css-text-901oao"
    dir="auto"
  >
    test
  </div>
</div>
`;
